numpy/_core/_add_newdocs.py

```diff
@@ -3619,6 +3619,10 @@
     access to elements of the array and doing arithmetic on elements of the
     array using Python's optimized math.
 
+    There are some edge cases when numpy scalar is returned. in this scenario,
+    `item` can be recursively called to get the standard python scalar. But for
+    `numpy.longdouble` type, `item` would return itself.
+
```

Comment:
IMO this feels like information overload and brings too much attention to a specific corner case. If anything, I'd vote for [@seberg's wording suggestion](https://github.com/numpy/numpy/issues/14139#issuecomment-515792885): 

> for an array with object dtype the elements are returned as-is

Reply:
I think it be good. But I have added some example codes, these information is to describe these examples.